src/constants/index.ts

```diff
@@ -34,6 +34,11 @@ export const STANDARD_TRANSFER_VALUE: BigNumber = new BigNumber(1000000000000000
 export const UNLIMITED_ALLOWANCE_IN_BASE_UNITS: BigNumber =
   SetProtocolUtils.CONSTANTS.UNLIMITED_ALLOWANCE_IN_BASE_UNITS; // 2 ** 256 - 1
 export const ZERO: BigNumber = SetProtocolUtils.CONSTANTS.ZERO;
+export const ONE: BigNumber = new BigNumber(1);
+export const TWO: BigNumber = new BigNumber(2);
+export const TEN: BigNumber = new BigNumber(10);
```

Comment:
I get what you're doing, it's awkward to define a big number in line and then use it immediately, but I wasn't sure what/why this constant existed. I'd actually prefer what we did in contracts and have an `ether(n)` function. Maybe something like `transactionUsableNumber(n)`, or maybe `uint256(n)`?